apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.tsx

```diff
@@ -74,7 +75,10 @@ const DisplayServiceAccountCard = (props: Props) => {
       setAdminApiError(undefined);
     } catch (e: any) {
       if (isApiErrorType(e)) handleApiError(e);
-      else throw new Error('Unhandled error exception');
+      else {
+        setUnknownError(e);
+        throw new Error('Unhandled error exception');
```

Comment:
I love having more error handling, but did you ever hit these in development?

If so we should probably try to know what happened ha. Otherwise, this works

Reply:
So this is some tribal knowledge I'll need to share with the team. So the way this works is that you'll notice in line 77. We run a check in the `isApiErrorType` function. What this does is basically tell us if this error is something we know about (ie Admin API failure, Data API Failure, bad credentials with the service keys etc...) So anything that we know about in the lambda, we will map it also in the client side here.

However, what about errors that we never accounted for, or things we never considered. They all go into this bucket, to which then we'll need to debug and add to our list of known errors described above. 

So the whole point of this is to capture the literally unknown errors we never accounted for. I guess I could print e.message in this case but hopefully you understand the greater architecture context.

Reply:
Yeah definitely, I figured this was to handle errors we haven't seen, I just saw this block twice and figured I'd ask